go.mod

```diff
@@ -291,6 +291,7 @@ replace (
 	github.com/Shopify/sarama => github.com/elastic/sarama v1.19.1-0.20220310193331-ebc2b0d8eef3
 	github.com/dop251/goja => github.com/andrewkroh/goja v0.0.0-20190128172624-dd2ac4456e20
 	github.com/dop251/goja_nodejs => github.com/dop251/goja_nodejs v0.0.0-20171011081505-adff31b136e6
+	github.com/kardianos/service => github.com/elastic/kardianos-service v0.0.0-20230215200102-9832e01049dd
```

Comment:
I think go.mod will allow a comment, if so can you please add one saying why we need to the replace directive for kardianos.